dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/JarLinkageReport.java

```diff
@@ -75,6 +75,11 @@ public String toString() {
     }
     for (LinkageErrorMissingField missingField : getMissingFieldErrors()) {
       builder.append(indent + missingField.getReference());
+      if (missingField.getTargetClassLocation() != null) {
```

Comment:
Could this formatting logic be pushed into a toString or other method on LinkageErrorMissingField? That would simplify this method and its tests.